gradle/libs.versions.toml

```diff
@@ -165,6 +165,7 @@ managed-netty-transport-native-epoll = { module = "io.netty:netty-transport-nati
 managed-netty-transport-native-kqueue = { module = "io.netty:netty-transport-native-kqueue", version.ref = "managed-netty" }
 managed-netty-transport-native-iouring = { module = "io.netty:netty-transport-native-io_uring", version.ref = "managed-netty" }
 managed-netty-transport-native-unix-common = { module = "io.netty:netty-transport-native-unix-common", version.ref = "managed-netty" }
+managed-netty-pkitesting = { module = "io.netty:netty-pkitesting", version.ref = "managed-netty" }
```

Comment:
does this need to be `managed` if it is only for testing?

Reply:
not necessarily, but since all the other netty deps are managed and users may want to use this module too, it seems reasonable to me